### PR TITLE
Fix/Update Documentation: Create and run Rocket.Chat SDK bot

### PR DESCRIPTION
**Pull request by bandantonio**
Example using [JS SDK](https://github.com/RocketChat/Rocket.Chat.js.SDK)

### CODE REVIEW

**Review by Sing-Li — APPROVED** [review]

File: bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+```
+npm init -y
+npm install @rocket.chat/sdk
+```
+
+**2. create bot files**
+
+To proceed with the simplest setup, you need to create two files: the first one will be
+responsible for working logic of the bot. Another one will contain a map with responses.
```
Comment (reetp) [inline bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md]: responsible "the" for working logic - map with "the" responses

File: bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    }
+}
+
+runbot()
+```
+
+**NOTE:** Adjust the `Environment Setup` section content to fit your server and user
+credentials.
+
+Make sure `<BOT USER NAME>` has `BOT` role on the server.
```
Comment (reetp) [inline bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md]: has "a" BOT role

File: bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+For more information about that, please refer to [this page](../#1-create-a-bot-user).
```
Comment (reetp) [inline bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md]: For more information on how to create a bot user please refer to [this page] ( blah)

File: bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+[getRoomIdByNameOrId] Success: "GENERAL"
+[sendMessage] Calling (async): [{"msg":"Rocket Chat BOT is listening ...","bot":{"i":"js.SDK"},"rid":"GENERAL"}]
+[sendMessage] Success: {"msg":"Rocket Chat BOT is listening ...","bot":{"i":"js.SDK"},"rid":"GENERAL","ts":{"$date":1573598821912},"u":{"_id":"GDJn6Exf9LpK2Pp5n","username":"botUser"},"mentions":[],"channels":[],"_updatedAt":{"$date":1573598821920},"_id":"6yTcbcNKuMDBdPsFc"}
+Greeting message sent
+[received] Message in room GENERAL
+```
+
+**4. talk to your bot**
+
+On the server, login as a regular user (not the BOT user), go to `general` room, and task to your newly created bot:
```
Comment (reetp) [inline bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md]: Not sure what this means??

" and task to your newly created bot:"

File: bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
+# Run a Rocket.Chat SDK Bot
+
+This bot is a simple example of how you can use [Rocket.Chat JS SDK](https://github.com/RocketChat/Rocket.Chat.js.SDK) methods directly. It
+does not handle errors, different messsage types, server resets and other production
```
Comment (reetp) [inline bots/create-and-run-a-bot/rocketchat-sdk-bot/README.md]: It is extremely basic (primitive/low level?) and does not handle.....

Or

Compared to other bots documented here it is extremely basic and does not handle blah

**Review by Sing-Li — APPROVED** [review]

**Pull request merged**